helen_susan_bejoy_app/add_books_feature/list_all_books.txt

```diff
@@ -0,0 +1 @@
+This is the feature which shows the user all the books they’ve created as a list.List all books will list all books in ascending order.
```

Comment:
Also add in the second sentence mentioned in task

Reply:
corrected